src/demos/calcite-link.html

```diff
@@ -33,9 +33,23 @@ <h3>Set focus</h3>
     <calcite-button onclick="document.querySelector('#button-focus-example-2').setFocus()">focus a </calcite-button>
     <h4>Receive focus</h4>
 
-    <calcite-link id="button-focus-example-1">span to gain focus</calcite-link>
-    <br />
-    <calcite-link id="button-focus-example-2" href="http://google.com">anchor to gain focus</calcite-link>
+
+  <h3>Text sizes</h3>
+  <p style="font-size:10px"><calcite-link icon-start="information" icon-end="information">example text</calcite-link><br /></p>
+  <p style="font-size:12px"><calcite-link icon-start="information" icon-end="information">example text</calcite-link><br /></p>
+  <p style="font-size:14px"><calcite-link icon-start="information" icon-end="information">example text</calcite-link><br /></p>
+  <p style="font-size:16px"><calcite-link icon-start="information" icon-end="information">example text</calcite-link><br /></p>
+  <p style="font-size:18px"><calcite-link icon-start="information" icon-end="information">example text</calcite-link><br /></p>
+  <p style="font-size:20px"><calcite-link icon-start="information" icon-end="information">example text</calcite-link><br /></p>
+  <p style="font-size:16px"><calcite-link icon-start="information" icon-end="information">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry' 1.10.32. The standard chunk of Lorem Ipsum used since the 1500s is reproduced below for those interested. Sections 1.10.32 and 1.10.33 from "de Finibus Bonorum et Malorum" by Cicero are also reproduced in their exact original form, accompanied by English versions from the 1914 translation by H. Rackham.</calcite-link><br /></p>
+
+
+  <h3>Set focus</h3>
```

Comment:
This needs cleanup. Can you change this back to how it looked before with calcite-links, under the "Receive focus" heading?